### ISSUE TITLE
VOR needles not showing

### ISSUE THREAD

**Issue by StephN-B** [flybywiresim/a32nx#5301]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-07-11T09:04:00+00:00",
  "ref": "master",
  "sha": "3c036f310c40996c8b7777359c17da6453e7d2d5",
  "actor": "davidwalschots",
  "event_name": "manual"
}
```

### Describe the bug

The needles/indicators for VOR1/VOR2 do not display in any of the Navigation screens. Distance to VOR stations is shown though.
NDB1/NDB2 constantly flicker.

### Expected behavior

- When the VOR station is in reach, the indicators need to point in the direction of the VOR
- NDB needles should be 'stable'

### Steps to reproduce

Can probably be recreated with any airport, but here are my steps:
- Create a flight from EDDM (Munich) to EDDH (Hamburg), departure runway 26R
- Place plane at any of the gates.
- Tune in the following stations:
VOR1: DMN (116.00)
VOR2: DMS (115.00)
NDB1: MNE (369)
NBD2: MSE (358)
- Taxi and take off
- You will see that at none of these stages the VOR needles show in any of the Navigation displays

**Comment by tracernz** [comment]
VOR needles should work okay, but the ADF station flicking is a known issue, fixed with the new ND in the upcoming custom flightplan manager branch.

**Comment by Venorcis** [comment]
I'm not sure if it's related and/or an Asobo bug, but some VORs seem to have very weird ranges. The AZR VOR at LFMN seems to cut out after 25NM already (see https://www.twitch.tv/videos/1116256719?t=01h49m20s), making it useless for executing the NERAS hold for which it is required by chart.

**Comment by tracernz** [comment]
> AZR

It is coded for 25 NM in the navigraph data. MSFS data (from NavBlue) codes it for 195 NM. You can open a request at the Navigraph forums.

**Comment by 2hwk** [comment]
Fixed by #4953 closing ☑️
